build/scripts/main.js

```diff
@@ -0,0 +1,83 @@
+$(document).ready(function() {
+    UpdateDateTime();
+});
+
+/*
+Load the date and populate the DOM.
+*/
+function LoadDate()
+{
+    // Date
```

Comment:
Don't need this comment. Doesn't add anything to the understanding

Reply:
Yeap, leftover comment from refactoring